src/docker.rs

```diff
@@ -87,6 +100,22 @@ pub fn run(target: &Target,
     let mount_root = mount_finder.find_mount_path(&root);
     let sysroot = mount_finder.find_mount_path(&sysroot);
 
+    docker_command("volume")?
+        .args(&["create", "data-volume"])
+        .run_and_get_status(verbose)?;
+
+    docker_command("create")?
+        .args(&["-v", "data-volume:/cross"])
+        .args(&["--name", "helper", "busybox", "sh", "-c", &format!("mkdir /cross/target && chown {user}:{grp} /cross/target && chown -R {user}:{grp} /cross/*", user = id::user(), grp = id::group())])
+        .run_and_get_status(verbose)?;
+
+    copy_files(&cargo_dir, "/cross", verbose)?;
+    copy_files(&xargo_dir, "/cross", verbose)?;
+    copy_files(&sysroot, "/cross", verbose)?;
+    copy_files(&mount_root, "/cross", verbose)?;
```

Comment:
Is there a solution other than copying every single file in our cargo, xargo, sysroot, and mount root over?